.github/ISSUE_TEMPLATE/sprint-release.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Sprint release
+about: Create a new sprint release
+title: Sprint release
+labels: 'Tracker,kind/documentation,area/release-eng'
```

Comment:
Is there any reason we assign labels differently for the other templates? This one looks pretty clean.

Reply:
no specific reason.
Included it.